aws/resource_aws_cognito_user_pool_client.go

```diff
@@ -141,6 +141,15 @@ func resourceAwsCognitoUserPoolClient() *schema.Resource {
 				},
 			},
 
+			"prevent_user_existence_errors": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "ENABLED",
```

Comment:
Since the [API has multiple defaults for this value](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_CreateUserPoolClient.html#CognitoUserPools-CreateUserPoolClient-request-PreventUserExistenceErrors), we should allow existing (`LEGACY`) configurations to not show a difference after update.

```suggestion
				Computed: true,
```